include/SFML/System/String.hpp

```diff
@@ -37,6 +37,46 @@
 
 namespace sf
 {
+////////////////////////////////////////////////////////////
+/// \brief Character traits for std::uint8_t
+///
+////////////////////////////////////////////////////////////
+struct SFML_SYSTEM_API U8StringCharTraits
+{
+    // NOLINTBEGIN(readability-identifier-naming)
```

Comment:
I don’t think we added clang-tidy to SFML 2 so we can probably remove these `NOLINT` comments.